src/browser/BrowserService.cpp

```diff
@@ -665,32 +835,32 @@ bool BrowserService::updateEntry(const QString& dbid,
         }
     }
 
-    QString username = entry->username();
+    auto username = entry->username();
     if (username.isEmpty()) {
         return false;
     }
 
     bool result = false;
-    if (username.compare(login, Qt::CaseSensitive) != 0
-        || entry->password().compare(password, Qt::CaseSensitive) != 0) {
+    if (username.compare(entryParameters.login, Qt::CaseSensitive) != 0
+        || entry->password().compare(entryParameters.password, Qt::CaseSensitive) != 0) {
         MessageBox::Button dialogResult = MessageBox::No;
         if (!browserSettings()->alwaysAllowUpdate()) {
             raiseWindow();
-            dialogResult = MessageBox::question(
-                m_currentDatabaseWidget,
-                tr("KeePassXC: Update Entry"),
-                tr("Do you want to update the information in %1 - %2?").arg(QUrl(siteUrl).host(), username),
-                MessageBox::Save | MessageBox::Cancel,
-                MessageBox::Cancel,
-                MessageBox::Raise);
+            dialogResult = MessageBox::question(m_currentDatabaseWidget,
+                                                tr("KeePassXC - Update Entry"),
+                                                tr("Do you want to update the information in %1 - %2?")
+                                                    .arg(QUrl(entryParameters.siteUrl).host(), username),
+                                                MessageBox::Save | MessageBox::Cancel,
+                                                MessageBox::Cancel,
+                                                MessageBox::Raise);
```

Comment:
This should be included: https://github.com/keepassxreboot/keepassxc/pull/9853

Reply:
Agreed